llvm/include/llvm/SYCLLowerIR/DeviceConfigFile.td

```diff
@@ -67,6 +67,7 @@ def AspectInt64_extended_atomics : Aspect<"int64_extended_atomics">;
 def AspectUsm_system_allocator : Aspect<"usm_system_allocator">;
 def AspectUsm_restricted_shared_allocations : Aspect<"usm_restricted_shared_allocations">;
 def AspectHost : Aspect<"host">;
+def AspectExt_oneapi_non_uniform_groups : Aspect<"ext_oneapi_non_uniform_groups">;
```

Comment:
Is it meant to be with the deprecated aspects? Functionally it makes no difference here, but it is a little confusing given line 64.

Reply:
Thanks for spotting this. I've moved it to the more appropriate place before the comment.